dog-blog.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+</head>
+<body>
+    <header>  
+    <img class="svg-dog-header" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo">
+    <p>Seattle's premiere dog adoption agency</p>
+    <div>
+      <img class="svg-cart" src="images/cart-icon.svg" alt="checkout cart icon">
+      <input class="cart-btn" type="submit" value="Checkout">
+    </div>
+    </header> 
+    <main>
+      <nav class="nav-head">
+        <a class="nav-item-head" href="index.html">Home<span class="space">/</span></a>
+        <a class="nav-item-head" href="dog-listing.html">Dogs<span class="space">/</span></a>
+        <a class="nav-item-head" href="contact.html">Contact</a>
+      </nav>
+      <h1 class="blog-adoptadog">Adoptadog Blog</h1>
+      <section>
+        <div>
+          <h2 class="blog-topic">Traveling With Your Dog</h2>
+          <img class="blog-image" src="images/blog-1.jpg" alt="A woman sitting looking out over the Grand Canyon with a dog">
```

Comment:
Would suggest putting your h2 and p tags into their own div. Then apply a CSS class to your post, and then to your text. I would suggest using flexbox to make these responsive.
```
<div class="blog-post">
   <img/>
   <div class="post-text">
      <h2></h2>
      <p></p>
   </div>
</div>
```